docs/source/installation/troubleshooting.rst

```diff
@@ -87,19 +89,19 @@ instructions given in
 of the corresponding `GitHub repository <https://github.com/ManimCommunity/ManimPango>`_.
 
 
-(Windows) OSError: dlopen() failed to load a library: pango?
+(Windows) OSError: dlopen() failed to load a library: pango
 ------------------------------------------------------------
```

Comment:
The underline below a section header should always have the same length as the title itself.

```suggestion
-----------------------------------------------------------
```